mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -362,8 +363,17 @@ def evaluate(self,
                 results, resfile_path, metrics)
             print_log('Evaluate CLEAR MOT results.', logger=logger)
             distth = 1 - track_iou_thr
-
             accs = []
+            # support loading data from ceph
+            client = self.file_client.client
```

Comment:
Directly using `self.file_client` rather than `self.file_client`.
`self.file_client` also provides `get()` API

Reply:
Okay, I will check it.